src/kudu/util/thread.cc

```diff
@@ -552,7 +552,7 @@ Status Thread::StartThread(const std::string& category, const std::string& name,
 
   {
     SCOPED_LOG_SLOW_EXECUTION_PREFIX(WARNING, 500 /* ms */, log_prefix, "creating pthread");
-    SCOPED_WATCH_STACK((flags & NO_STACK_WATCHDOG) ? 0 : 250);
+    //SCOPED_WATCH_STACK((flags & NO_STACK_WATCHDOG) ? 0 : 250);
```

Comment:
Did you run into any perf issues with the watchdog?
They might have fixed it in a later revision: http://mail-archives.eu.apache.org/mod_mbox/kudu-commits/201610.mbox/%3C16d2131cc82e47f0ab4638829cca1d54@git.apache.org%3E

Reply:
Kernel Stack Watchdog thread was causing the crashes in uninstall plugin path. The RPC worker threads and supervisor threads would crash intermittently due to this feature.